preprocessing/creatingData/split_video.py

```diff
@@ -0,0 +1,85 @@
+# DOCUMENTATION
+#   Purpose: Break video files into seperate frames and label it
+#   Note: This combines both the video_frames.py and the label_frames.py scripts
+# Command Line Input:
+#   python video_frames.py  --directory <directory path to store frames> --video <path to video file> --file <path to timestamp file>
+#   EXAMPLE:
+#   python video_frames.py --directory ./underwater-video3-frames --video ./underwater_footage/video1.mp4 --file ./test.csv
+# Optional Arguments:
+#   --compress: optionally compress image after saving
+
+import cv2
+import os
+import argparse
+
+from PIL import Image
+
+# Construct the argument parser
+ap = argparse.ArgumentParser()
+ap.add_argument("-v", "--video", required=True,
+    help="path to the video file")
+ap.add_argument("-d", "--directory", required=True,
+    help="path to the directory to store the frames. Will create a new directory \
+    if needed")
+ap.add_argument("-f", "--file", required=True,
+    help="path to the text file containing timestamps for objects in video")
+ap.add_argument("-c", "--compress", action="store_true",
+    help="compress images before saving")
+args = vars(ap.parse_args())
+
+def getFrame(time, framerate):
```

Comment:
Can you add docstrings to functions? ie
def getFrame(time, framerate):
   """
   A function to get the frame number from a given time and framerate
   inputs:
   time: the time as a string, in format hh:mm:ss
   framerate: the framerate as a float
   outputs:
   the frame number as an integer
   """